questions.md

```diff
@@ -132,5 +131,13 @@ Please keep in mind that this document is in a continuous state of flux, with ne
 - Have you heard of or used `Cocoapods`, `Carthage`, and the `Swift Package Manager`?
 - Which version control tools do you use? How much experience do you have with `git`? Can you explain what a `rebase` operation is?
 
+### Computer Science 101 (Questions + Knowledge)
+
+- What is a `deadlock`? How do you avoid it?
+- What is a `race condition`? How do you avoid it?
+- Knowledge about Big O notation
+- Knowledge about data structures: graphs, trees, queues, stacks, linked lists, hashtables and others. 
+- Knowledge about algorithms: sorting, searching and others. 
```

Comment:
These sound a lot like whiteboard-y questions :S

Reply:
Well I think it depends on the position. If we're aiming for seniors okay, for lower I think you can get far without them.

It all boils down to who (as in candidate) you are okay to filter out and possibly lose. If you are okay with that go ahead.